tests/e2e/actions.go

```diff
@@ -1819,8 +1819,9 @@ type assignConsumerPubKeyAction struct {
 func (tr TestRun) assignConsumerPubKey(action assignConsumerPubKeyAction, verbose bool) {
 	valCfg := tr.validatorConfigs[action.validator]
 
+	// Note: to get error response reported back from this command '--gas auto' needs to be set.
```

Comment:
I don't find this comment satisfying, but it's definitely better than nothing.
Is it that the gas limit that was there before was somehow not enough?
Not sure if you want to dive in deeper to find out, if not I think the current comment is good enough to warn us to be careful about changing this.

Reply:
agree. gas-limit as root cause is an assumption we had but this is not yet identified/confirmed.
I'll keep it for now and update in case I find some time for  some bi-secting runs to narrow down the issue

Reply:
Sounds reasonable. I wouldn't fuss too much about it, definitely shouldn't block this pr imo